src/components/VHeader/VHeaderMenu.vue

```diff
@@ -10,7 +10,7 @@ import {
   useRouter,
 } from '@nuxtjs/composition-api'
 
-import { ALL_MEDIA, searchPath, supportedMediaTypes } from '~/constants/media'
+import { ALL_MEDIA, searchPath, SupportedMediaType } from '~/constants/media'
```

Comment:
Code editor alerts on line 38:
> const isMinScreenMd: Ref<boolean>
Type 'Ref<boolean> | undefined' is not assignable to type 'Ref<boolean>'.
  Type 'undefined' is not assignable to type 'Ref<boolean>'.Vetur(2322)

Reply:
Great catch! The `inject` falls back to undefined if the value is not provided. I'll set a default to `ref(false)`.

Reply:
@krysal, are you using Vetur extension in VSCode, not Volar?